cdflow_commands/account.py

```diff
@@ -9,6 +10,20 @@ def __init__(self, alias, id, role):
         self.role = role
 
 
+def replace_team(scheme, team):
+    if type(scheme) is str:
+        return re.sub(r'{team}', team, scheme)
```

Comment:
Why use a regular expression here rather than Python string formatting? e.g. `return scheme.format(team=team)`

Reply:
because it means you have to escape other curlies - was thinking of this as just a `{team}` placeholder, rather than treating every string as a python template with required escaping - does that make sense?

Reply:
Yeah, that makes sense.